controllers/UserOAuth.go

```diff
@@ -0,0 +1,66 @@
+
+// MentorOauth Handler for Github OAuth of Mentor
+func MentorOAuth(w http.ResponseWriter, r *http.Request) {
```

Comment:
Function name `MentorOAuth` 
This is common for both right? Better something like UserOAuth